tests/framework/jailer.py

```diff
@@ -103,7 +102,7 @@ def construct_param_list(self):
         if self.chroot_base is not None:
             jailer_param_list.extend(["--chroot-base-dir", str(self.chroot_base)])
         if self.netns is not None:
```

Comment:
Either we should make `netns` optional on the constructor or this if is not needed, right?

Reply:
Ah yeah, that's a negative test. Actually we don't need to pass None explicitly now. That could be removed too.